docs/root/configuration/listeners/network_filters/wasm_filter.rst

```diff
@@ -23,15 +23,14 @@ Example filter configuration:
   name: envoy.filters.network.wasm
   typed_config:
     "@type": type.googleapis.com/envoy.extensions.filters.network.wasm.v3.Wasm
-    config:
-      config:
-        name: "my_plugin"
-        vm_config:
-          runtime: "envoy.wasm.runtime.v8"
-          code:
-            local:
-              filename: "/etc/envoy_filter_http_wasm_example.wasm"
-          allow_precompiled: true
+     config:
```

Comment:
hi @hanxiaop thanks for working on this

with configs in the docs we are moving them to standalone bootstrap yaml files so that they can be validated and tested

would you be able to move this one to a boostrap config ?

if you do `git grep literalinclude` you should be able to find existing examples

Reply:
@phlax Thanks for reminding. I've revised to a valid standalone config, but however, I think the config cannot pass CI since it needs a `*.wasm` file, and it looks like the CI system supports multiarch. Do you have any suggestions?

Reply:
if you check existing configs for wasm, i think there is a wasm bin in the test build - that should work i think

(see `examples/wasm-cc/envoy.yaml`)

re arch support - i would look how the existing wasm config test is restricted - see `examples/wasm-cc/BUILD`